### PR TITLE
Asset collections graph data should now be sorted and combined

### PR DESCRIPTION
**Pull request by LefterisJP**
Also added a unit test for this

### CODE REVIEW

File: rotkehlchen/db/dbhandler.py [hunk 1]
```diff
@@ -2220,6 +2220,7 @@ def query_collection_timed_balances(
                     to_ts=to_ts,
                 ))
 
+        asset_balances.sort(key=lambda x: x.time)
```
Comment (LefterisJP) [inline rotkehlchen/db/dbhandler.py:2223]: this is the one line fix. Rest is to test

**Review by yabirgb — APPROVED** [review]
lgtm

**Comment by LefterisJP** [comment]
Seems that `rotkehlchen/tests/api/test_bitcoin.py::test_add_xpub_with_conversion_works` fails quite often lately. We should find out why.

**Comment by LefterisJP** [comment]
Anyway will merge, and may look at it later

**Pull request merged**